README.md

```diff
@@ -8,34 +8,318 @@ or
+```
+import React, {Component} from 'react'
+
+import { BuyProduct, CartButton, Footer } from 'teepublic-react'
+// We will talk about this in Step 9
+import { TeePublicRoutes } from '../TeePublicConfiguration'
+
+export default class TeePublicDesign extends Component {
   render () {
-    const avatarUrl = this.props.store_data.avatarUrl
-    const bannerUrl = this.props.store_data.bannerUrl
-    const designs   = this.props.store_data._embedded.designs
-    const onClickHandler   = (design, sku) => {
-      console.log(design);
-      console.log(sku);
-    }
+    const { design, store, skuData, affiliateNetworkId } = this.props
 
     return (
-      <div className='homepage'>
-        <img src={bannerUrl}/>
-        <br/>
-        <img src={avatarUrl}/>
-
-        <DesignCollection
-          designs={designs}
-          tileSize="large"
-          onDesignClick={ onClickHandler }
+      <div className='teepublic design'>
+        <CartButton className='contain' href={TeePublicRoutes.cartUrl()} onCheckout={TeePublicRoutes.onCheckout} />
+
+        <BuyProduct
+          design={design}
+          store={store}
+          skuData={skuData}
+          storeUrl={TeePublicRoutes.storeUrl}
+          buyProductLinkBuilder={TeePublicRoutes.buyProductUrl}
+          tagLinkBuilder={TeePublicRoutes.tagUrl}
+          affiliateNetworkId={affiliateNetworkId}
+          className='contain'
         />
 
+        <Footer />
       </div>
-    );
+    )
+  }
+}
+```
+
+#### `<TeePublicCart />`
+Add this new component to your react component library.
+Copy paste the code below with your necessary tweaks
+```
+import React, {Component} from 'react'
+
+import { Cart, Footer } from 'teepublic-react'
+// We will talk about this in Step 9
+import { TeePublicRoutes } from '../TeePublicConfiguration'
+
+export default class TeePublicCart extends Component {
+  render () {
+    return (
+      <div className='teepublic cart'>
+        <Cart
+          className='contain'
+          onCheckout={TeePublicRoutes.onCheckout}
+          storePathLinkBuilder={TeePublicRoutes.storesUrl}
+          buyProductLinkBuilder={TeePublicRoutes.buyProductUrl}
+        />
+
+        <Footer />
+      </div>
+    )
+  }
+}
+```
+
+## Step 9 - Understanding Route Builders
+Each application can host teepublic store, product or cart pages according to their routing convention.
+For eg.
+https://website.com/stores/store_id
+https://website.com/cart
+https://website.com/design/design_id
+
+React components allows applications to pass functions which knows how to build your application specific routes.
+It would be a good practice if application maintains these route builders in one file.  
+
+Create a `TeepublicConfiguration.js` in your javascript folder and copy paste the content below and change the functions according to your application URLs for TeePublic pages. Keep the `tagUrl` and `onCheckout` function as it is for now and they will just work without any change.
```

Comment:
Move the "Keep `tagUrl and `onCheckout`" notes into the code.

Reply:
Understood

Great review